### PR TITLE
Attach to tab before accessing stylesheets object - fixes #1

### LINKED ISSUE #1: It doesn't work with 2.1 and 2.2 simulators
There is a what I think is the same bug filed here: [FirefoxOS simulator 2.2 StyleSheets getStyleSheets throw error from remote debugger](https://bugzilla.mozilla.org/show_bug.cgi?id=1071711) but the paste bin has expired and I can't see if it's the same error.

We need to write a smaller test case that shows the error so devtools people can debug it and see what's going on-the test pseudocode would be:
- launch simulator
- connect to obtain client
- launch app
- get app tab/actor using the client
- see how `tab.StyleSheets.getStyleSheets` never calls the callback in 2.1 or 2.2, but does in 2.0

Essentially the conflicting code is [the code of this module](https://github.com/mozilla/node-firefox-reload-css/blob/4348416afe74c631d3c60c2dc756093463161657/index.js#L24-L29)

### PR DESCRIPTION
**Pull request by sole**
fix for #1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,12 +26,17 @@ module.exports = function reloadCSS(options) {
...
       .then(function(tab) {
-        tab.StyleSheets.getStyleSheets(function(err, styleSheets) {
-          if (err) {
-            reject(err);
-          }
-          var appOrigin = app.origin;
-          resolve(updateStyleSheets(styleSheets, appOrigin, appSrcPath));
+        // Apparently this method needs to be called in tabs before you can
+        // do anything at all on them. This is a change introduced in simulators
+        // 2.1 onwards.
```
Comment (tofumatt) [inline index.js:31]: It doesn't break earlier-than-2.1 devices though, right?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-firefox-reload-css",
-  "version": "1.1.1",
+  "version": "1.2.1",
```
Comment (tofumatt) [inline package.json]: Hehe, shouldn't it really be `1.1.2`? Or even `1.2.0`? :grin:

File: index.js (hunk 1 above)
Comment (sole) [inline index.js:31]: '2.1 onwards'

File: package.json (hunk 2 above)
Comment (sole) [inline package.json]: 1.2.0, I'll fix that.

File: index.js (hunk 1 above)
Comment (tofumatt) [inline index.js:31]: I just meant "do tabs before this version have that method?" but I will assume yes.

**Pull request merged**